index.Rmd

```diff
@@ -95,14 +95,14 @@ roads_water_plot
 ```
 
 
-# Results
+# Results 
 
 This is where the results of the gdistance tests will be displayed in table format
 
-# Conclusions
+# Conclusions 
```

Comment:
Depending on what your results show, it might be useful to identify limitations or other behavioral considerations (do people always take the shortest route on the roads?  Or might they cut across non-road terrain, perhaps less likely if traveling via carriage or with supplies.)  This may not be all that important for this project but perhaps something to consider if it's part of a larger study.